GliaWidgetsTests/Sources/CallViewControllerTests.swift

```diff
@@ -0,0 +1,27 @@
+@testable import GliaWidgets
+import SalemoveSDK
+import XCTest
+
+class CallViewControllerTests: XCTestCase {
+
+    func test_call_deinit() {
+
```

Comment:
Also here. And similar thing happens in the texts below

Reply:
Fixed [6c181fd](https://github.com/salemove/ios-sdk-widgets/pull/416/commits/6c181fde74064d1185ae3311815fcd2645dc12b3)